File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 6 and 7 July 2019/1.PoolDay/1.PoolDay.js
function poolDay(arg) {
  people = +arg[0]
  tax    = +arg[1]
  shezlong = +arg[2]
  umbrela  = +arg[3]

  enterTax = people * tax  //115.50
  precentShezlong = (Math.ceil(people  * 0.75) * shezlong).toFixed(2) //70.40
  precetnUmbrela =  (Math.ceil(people * 0.50) * umbrela).toFixed(2)   //68.20

  totalPrice = (+enterTax + +precentShezlong + +precetnUmbrela).toFixed(2)
  final = totalPrice + ' lv.'

  console.log(final)
}

//poolDay(['100','8','6','4',])

File: Python Exams/01. Basic Exam Collection/Basics Online Exam - 6 and 7 July 2019/1.PoolDay/1.PoolDay.py
from math import ceil
people, tax, sunbed, umbrella = [float(input()) for _ in range(4)]

enter_tax = people * tax 
percent_sunbed = ceil(people * 0.75) * sunbed
percent_umbrella =  ceil(people * 0.50) * umbrella
total_price = enter_tax + percent_sunbed + percent_umbrella

print(f'{total_price:.2f} lv.')


'''
21
5.5
4.4
6.2
'''
